frontend.bat

```diff
@@ -523,14 +523,18 @@ for /F "tokens=1,2 delims=." %%a in ("!vsver!") do (
     rem https://github.com/3F/hMSBuild/issues/3
     set vsver=%%~a.0
 )
+REM Workaround for VS2019, may use 'vswhere -find' instead
+if !vsver! geq 16.0 set vsver=Current
 set _msbp=!vspath!\MSBuild\!vsver!\Bin
 
 call :dbgprint "found path via vswhere: " _msbp
 
+if not defined notamd64 (
```

Comment:
well, seems it may increase some performance. Thanks. 

How about single line without braces `()` ? ie. serial if ... if ... means `AND` logic.

```bat
if not defined notamd64 if exist "!_msbp!\amd64" (
  ...
)
```

Also: 

This logic from **1.x versions**. As I remember **only for** normalization of the paths to x64 build. Because amd64 is default path for 64bit target machines. here's [final -notamd64 filter](https://github.com/3F/hMSBuild/blob/a40c6ac420708cbc86a2d93c9f374feeb42bb9cd/frontend.bat#L599)

But today for **2.x** I don't remember, does this normalization really need together with vwhere at all -_-

Reply:
if you're also changing this, can you also review necessity of this block at all? something bothers me. Thanks.

Reply:
Removed it, was never actually needed